packages/material-ui/src/Stepper/Stepper.test.js

```diff
@@ -37,143 +35,184 @@ describe('<Stepper />', () => {
   );
 
   it('has no elevation by default', () => {
-    const wrapper = mount(
+    const { container } = render(
       <Stepper>
         <Step />
       </Stepper>,
     );
-    expect(wrapper.find(Paper).props().elevation).to.equal(0);
+
+    const paper = container.querySelector('.MuiPaper-elevation0');
+
+    expect(paper).not.equal(null);
   });
 
   describe('rendering children', () => {
     it('renders 3 Step and 2 StepConnector components', () => {
-      const wrapper = mount(
+      const { container } = render(
         <Stepper>
           <Step />
           <Step />
           <Step />
         </Stepper>,
       );
 
-      expect(wrapper.find(StepConnector).length).to.equal(2);
-      expect(wrapper.find(Step).length).to.equal(3);
+      const connectors = container.querySelectorAll('.MuiStepConnector-root');
+      const steps = container.querySelectorAll('.MuiStep-root');
+
+      expect(connectors).to.have.length(2);
+      expect(steps).to.have.length(3);
     });
   });
 
   describe('controlling child props', () => {
     it('controls children linearly based on the activeStep prop', () => {
-      const wrapper = shallow(
+      const { container, setProps } = render(
         <Stepper activeStep={0}>
-          <div className="child-0" />
-          <div className="child-1" />
-          <div className="child-2" />
+          <Step />
+          <Step />
+          <Step />
         </Stepper>,
       );
-      expect(wrapper.find('.child-0').props().active).to.equal(true);
-      expect(wrapper.find('.child-1').props().active).to.equal(false);
-      expect(wrapper.find('.child-2').props().active).to.equal(false);
-      expect(wrapper.find('.child-1').props().disabled).to.equal(true);
-      expect(wrapper.find('.child-2').props().disabled).to.equal(true);
-      wrapper.setProps({ activeStep: 1 });
-      expect(wrapper.find('.child-0').props().completed).to.equal(true);
-      expect(wrapper.find('.child-0').props().active).to.equal(false);
-      expect(wrapper.find('.child-1').props().active).to.equal(true);
-      expect(wrapper.find('.child-2').props().active).to.equal(false);
-      expect(wrapper.find('.child-2').props().disabled).to.equal(true);
+
+      const steps = container.querySelectorAll('.MuiStep-root');
+      const connectors = container.querySelectorAll('.MuiStepConnector-root');
+
+      expect(steps[0]).to.not.have.class('MuiStep-completed');
```

Comment:
We don't need to hardcode these. You can get the classnames of `Step` with `const stepClasses = getClasses(<Step />)`. We should already be doing this for the Stepper classes in this test.

Reply:
But `getClasses` uses `createShallow` which is enzyme API. I thought we want to replace enzyme with testing-library

Reply:
One at a time. We still use `getClasses` in all unit test and will likely continue. Can't drop enzyme completely anyway.

Reply:
Done. And for the other tests too.